django/db/backends/sqlite3/base.py

```diff
@@ -40,6 +40,12 @@ def adapt_datetime(val):
     return val.isoformat(" ")
 
 
+def _get_varchar_column(data):
+    if data["max_length"] is None:
+        return "varchar"
+    return "varchar(%(max_length)s)" % data
```

Comment:
Ahh this is the part I was missing!

Reply:
Thank you for the excellent guidance, which allowed me to proceed quickly.